front/app/modules/commercial/analytics/admin/components/VisitorsLanguageCard/index.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useRef } from 'react';
+
+// hooks
+import useVisitorsLanguageData from '../../hooks/useVisitorsLanguageData';
+
+// components
+import GraphCard from 'components/admin/GraphCard';
+import PieChart from 'components/admin/Graphs/PieChart';
+import { Box } from '@citizenlab/cl2-component-library';
+import EmptyState from 'components/admin/Graphs/_components/EmptyState';
+
+// i18n
+import messages from './messages';
+import { injectIntl } from 'utils/cl-intl';
+import { InjectedIntlProps } from 'react-intl';
+
+// typings
+import { IResolution } from 'components/admin/ResolutionControl';
+import { Moment } from 'moment';
+import { isNilOrError } from 'utils/helperUtils';
+import { LegendItem } from 'components/admin/Graphs/_components/Legend/typings';
+
+interface Props {
+  startAtMoment: Moment | null | undefined;
+  endAtMoment: Moment | null;
+  projectFilter: string | undefined;
+  resolution: IResolution;
+}
+
+const VisitorsCard = ({
+  startAtMoment,
+  endAtMoment,
+  projectFilter,
+  resolution,
+  intl: { formatMessage },
+}: Props & InjectedIntlProps) => {
+  const graphRef = useRef();
+
+  const { pieData, xlsxData } = useVisitorsLanguageData();
+
```

Comment:
There is a more elaborate empty state design: https://www.figma.com/file/MySXSuHane4dc5CMSIBOGk/%F0%9F%93%8ABetter-Dashboards?node-id=2712%3A60464